src/components/technologies/genericTechnologies/stor.py

```diff
@@ -385,7 +399,6 @@ def bind_disjunctions(dis, t):
 
         # Maximal charging and discharging rates
         def init_maximal_charge(const, t):
-            # input[t] <= chargeCapacity
```

Comment:
i think luca/inge added these lines on purpose, maybe check with them before deleting?

Reply:
I decided to remove them from the pyomo rules as all equations are documented in the docstring.

Reply:
I agree, but might be nice to check with them since they did the effort of adding them. I remember that either luca or inge mentioned they thought it was much easier to read this way

Reply:
I checked with Luca.